azure-pipelines.yml

```diff
@@ -31,4 +31,10 @@ steps:
     SERVER_PASSWORD: '$(APPLICATION_SECRET)'
   inputs:
     command: 'custom'
-    customCommand: 'test'
\ No newline at end of file
+    customCommand: 'test'
+- task: Docker@2
+  inputs:
+    command: 'build'
+    Dockerfile: '**/Dockerfile'
+    buildContext: '**'
+    tags: 'WCB:$(Build.BuildId)'
```

Comment:
minor comment:

we can leverage versioning here:
https://github.com/AgaKhanFoundation/WCB/blob/master/Makefile#L3

Reply:
Sure, happy to change that in a follow up - but, can we do that without the unix commands?  Furthermore, using make obscures the logs and makes it harder to tell what is going on.